README.md

```diff
@@ -27,7 +30,7 @@ To use this action, simply add a step to your workflow, like so:
 ```
 
 Here’s an example workflow that checks out your repo and checks that you have
-the correct lockfile for Yarn:
+the correct lockfile for Yarn within a custom theme for a Drupal project:
```

Comment:
Maybe avoid the mention of Drupal? I'm concerned someone might think this only works for Drupal. I understand that this is just an example, but perhaps best to keep it generic.

Reply:
I was on the fence about it, but I think you’re right. Thanks!

Reply:
See 45650b4.